specification/sql/resource-manager/Microsoft.Sql/preview/2017-10-01-preview/databases.json

```diff
@@ -787,7 +787,7 @@
           "type": "boolean"
         },
         "licenseType": {
-          "description": "The license type to apply for this database.",
+          "description": "The license type to apply for this database. `LicenseIncluded` if you need a license, or `BasePrice` if you have a license and are eligible for the Azure Hybrid Benefit.",
```

Comment:
I think we usually use '' instead of `` in description.

Reply:
@fengzhou-msft - I'm not seeing any instances of 'instead of' in other descriptions that describe selecting from multiple options.  Is the updated description not clear? We're open to suggestions.

+ @vitomaz-msft - who requested the change.

Reply:
I mean use 'LicenseIncluded' instead of \`LicenseIncluded\` in description.